recipe/meta.yaml

```
@@ -52,15 +52,18 @@ requirements:
     - librsvg     2.54.4          # [osx]
     - libtheora   1.1.1
     - libvorbis   1.3.7           # [not win]
-    - libxml2     {{ libxml2 }}
+    - libxml2     2.10
```

```suggestion
    - libxml2     {{ libxml2 }}
```
2.13 is the version pinned to our cbc.

---

`2.13` creates conflicts with other packages, hence the manual pinning to `2.10`.

---

Well that's not the way to solve this, we switched to 2.13 as the base version. Do you have a list of these other packages?

---

An error on `osx-64` was:
`- package fontconfig-2.14.1-h1659381_0 requires libxml2 >=2.9.14,<2.10.0a0, but none of the providers can be installed`
But the `fontconfig`'s latest build number should be `3` not `0`, see https://github.com/AnacondaRecipes/fontconfig-feedstock/blob/master/recipe/meta.yaml#L15

---

> Well that's not the way to solve this, we switched to 2.13 as the base version. Do you have a list of these other packages?

I understand but this is a rebuild. I thought since it was already built with `2.10` it could be rebuilt with the same version.

---

You would typically rebuild with the newer version (the current one pinned in the cbc). And in some cases have yet another rebuild with an older version, but only when justified.
It looks like ffmpeg was missed due to being released around the same time as the libxml2 rebuild.